exe/make_a_round.cc

```diff
@@ -0,0 +1,904 @@
+#include <thread>
```

Comment:
This code looks like the simulator itself involved with some other exes. Why do you implement it again in the exe?